src/main/java/org/openstreetmap/atlas/checks/utility/AtlasToOsmType.java

```diff
@@ -0,0 +1,47 @@
+package org.openstreetmap.atlas.checks.utility;
+
+import java.util.EnumMap;
+
+import javax.annotation.Nullable;
+
+import org.openstreetmap.atlas.checks.validation.relations.InvalidMultiPolygonRelationCheck;
+import org.openstreetmap.atlas.geography.atlas.items.ItemType;
+
+/**
+ * Convert Atlas ItemTypes into OSM types
+ *
+ * @author Taylor Smock, extracted from {@link InvalidMultiPolygonRelationCheck}.
+ */
+public final class AtlasToOsmType
```

Comment:
It probably could be. I think I didn't do that since I've got a few projects with various moving parts, and I didn't want to deal with yet another one at the time, and then forgot about it.

I've had too many PR's that depend upon other PR's that depend upon yet another PR...

Reply:
This can defiantly be something that is done later.

Reply:
Yp!!